tests/Gaufrette/Functional/Adapter/FunctionalTestCase.php

```diff
@@ -67,8 +67,6 @@ public function shouldWriteAndRead()
 
         $this->assertEquals('Some content', $this->filesystem->read('foo'));
         $this->assertEquals('Some content1', $this->filesystem->read('test/subdir/foo'));
-        $this->filesystem->delete('foo');
```

Comment:
Are the removal to the ` $this->filesystem->delete` calls mandatory ? (because it also affects the other adapter tests)